### PR TITLE
Integrating changing themes on Settings page

### PR DESCRIPTION
**Pull request by baja156**
**Implementing theme changes on settings page**
- [ ] The pull request is complete according to the following criteria:
  - [ ] Acceptance criteria have been met
  - [ ] The documentation is kept up-to-date
  - [ ] Comprehensive tests (if applicable) have been generated and all pass.
  - [ ] The pull request describes the changes that have been made, and enough information is present in the description for any developer to understand what has changed
  - [ ] Commits have been squashed (or will be on merge).
  - [ ] The branch name is descriptive and follows the pull request title format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The pull request title is of the following format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The description uses [github syntax](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to link to the issue. E,g Resolves se701g2/Doto#{Number}
  - [ ] At least two reviewers assigned. One of which must be the assigner of the issue.
  - [ ] If there are merge conflicts, run git rebase as opposed to git merge with master.

### CODE REVIEW

**Comment by res550** [comment]
Will this change the theme on all of the pages?

**Comment by baja156** [comment]
> Will this change the theme on all of the pages?

Currently only on Settings page

File: doto-frontend/src/components/pages/Calendar/Calendar.js [hunk 1]
```diff
@@ -12,8 +12,10 @@ import CalendarTodayIcon from "@material-ui/icons/CalendarToday";
 import AddIcon from "@material-ui/icons/Add";
 import Tooltip from "@material-ui/core/Tooltip";
 import Header from "../Header";
-
 import "./Calendar.css";
+import "../Pages.css";
+
+// const classnames = require("classnames");
```
Comment (lakshyaoberoi) [inline doto-frontend/src/components/pages/Calendar/Calendar.js]: Could you please clean up the comments before merging?

File: doto-frontend/src/components/pages/Settings/SettingsPage.js [hunk 2]
```diff
@@ -136,12 +123,40 @@ const SubmitButton = () => {
 };
 
 const SettingsPage = () => {
+    const [theme, setTheme] = useState("blue");
```
Comment (lakshyaoberoi) [inline doto-frontend/src/components/pages/Settings/SettingsPage.js]: Will this be used globally - can this be called by other functions? Or does this need right up at the top?

**Comment by lakshyaoberoi** [comment]
Good job on splitting out the background into a separate file. It will definitely be easier to re-use!

File: doto-frontend/src/components/pages/Settings/SettingsPage.js (hunk 2 above)
Comment (baja156) [reply doto-frontend/src/components/pages/Settings/SettingsPage.js]: No, at the moment themes only work with the settings page

**Pull request merged**